src/displayapp/screens/settings/SettingChimes.h

```diff
@@ -23,16 +23,14 @@ namespace Pinetime {
           Controllers::Settings::ChimesOption chimesOption;
           const char* name;
         };
-        static constexpr std::array<Option, 3> options = {{
-          {Controllers::Settings::ChimesOption::None, " Off"},
-          {Controllers::Settings::ChimesOption::Hours, " Every hour"},
-          {Controllers::Settings::ChimesOption::HalfHours, " Every 30 mins"}
-        }};
+        static constexpr std::array<Option, 3> options = {{{Controllers::Settings::ChimesOption::None, "Off"},
+                                                           {Controllers::Settings::ChimesOption::Hours, "Every hour"},
+                                                           {Controllers::Settings::ChimesOption::HalfHours, "Every 30 mins"}}};
 
         std::array<lv_obj_t*, options.size()> cbOption;
 
         Controllers::Settings& settingsController;
       };
     }
   }
-}
+}
```

Comment:
Does the clang format actually require no newline at the end of the file? I don't think so, could you please check and revert

```suggestion
}

```